corehq/apps/reports/standard/project_health.py

```diff
@@ -235,7 +235,7 @@ def get_newly_performing(self):
         if self._previous_summary:
             dropouts = filter(
                 lambda stub: stub.is_newly_performing and not
-                CommCareUser.get(stub.user_id).is_deleted(),
+                CommCareUser.get(stub.user_id).is_deleted() and CommCareUser.get(stub.user_id).is_active,
```

Comment:
This implementation has some performance concerns.  Each of these calls will perform a database round trip, so that's 6 round trips to the db for each user in the domain.  Conceptually the `is_deleted()` and `is_active` checks feel like they should happen in the `get_all_user_stubs_with_extra_data` method, and that will also make it easier to do this in bulk.

Bulk options are couch (maybe) and elasticsearch.  We can discuss what that'd look like tomorrow, if you like.